### PR TITLE
Update: new Statistic entities

### PR DESCRIPTION
**Pull request by ntsanq**

### CODE REVIEW

File: src/Repository/RoutesStatisticRepository.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+/**
+ * @extends ServiceEntityRepository<RoutesStatistic>
+ *
+ * @method RoutesStatistic|null find($id, $lockMode = null, $lockVersion = null)
+ * @method RoutesStatistic|null findOneBy(array $criteria, array $orderBy = null)
+ * @method RoutesStatistic[]    findAll()
+ * @method RoutesStatistic[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class RoutesStatisticRepository extends ServiceEntityRepository
```
Comment (tolehoai) [inline src/Repository/RoutesStatisticRepository.php]: It should be extend BaseRepository

File: src/Repository/RoutesStatisticRepository.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    {
+        $this->getEntityManager()->remove($entity);
+
+        if ($flush) {
+            $this->getEntityManager()->flush();
+        }
+    }
+
+//    /**
+//     * @return RoutesStatistic[] Returns an array of RoutesStatistic objects
```
Comment (tolehoai) [inline src/Repository/RoutesStatisticRepository.php]: remove this comment

File: src/Repository/TicketsStatisticRepository.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    public function remove(TicketsStatistic $entity, bool $flush = false): void
+    {
+        $this->getEntityManager()->remove($entity);
+
+        if ($flush) {
+            $this->getEntityManager()->flush();
+        }
+    }
+
+//    /**
```
Comment (tolehoai) [inline src/Repository/TicketsStatisticRepository.php]: remove this

**Review by tolehoai — APPROVED** [review]

**Review by Ductuan92 — APPROVED** [review]
your code look good

**Pull request merged**